_includes/head.html

```diff
@@ -13,6 +13,12 @@
   gtag('config', '{{ site.ga_token }}');
 
         </script>
+        {% if page.schemadotorg %}
+        <script type="application/ld+json">
+          {% assign var = page.schemadotorg %}
+          {{ site.data[var] | jsonify}}
```

Comment:
This is pretty super long when inspecting the source. Perhaps pipe to `pretty_json` as well?

Reply:
After a bit of digging there does not seem to be any built-in `pretty_json` filter. Possible something to capture as an extension? Or rely on tooling like https://search.google.com/structured-data/testing-tool/ to expose the JSON in a more parsable way?